packs/data/feats.db/strix-vengeance.json

```diff
@@ -13,7 +13,7 @@
             "value": null
         },
         "description": {
-            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<p><strong>Trigger</strong> You, or a strix ally you can see, are damaged by an enemy's critical hit.</p>\n<hr />\n<p>You dedicate yourself to destroying those who harm your kin. Until the end of your next turn, you deal an additional +1d6 damage on Strikes against the triggering enemy. The bonus increases to +2d6 if you use a @Compendium[pf2e.equipment-srd.Striking]{Striking} weapon or unarmed attack and +3d6 if you use a @Compendium[pf2e.equipment-srd.Striking (Major)]{Striking (Major)} weapon or unarmed attack.</p>"
+            "value": "<p><strong>Frequency</strong> once per 10 minutes</p>\n<p><strong>Trigger</strong> You, or a strix ally you can see, are damaged by an enemy's critical hit.</p>\n<hr />\n<p>You dedicate yourself to destroying those who harm your kin. Until the end of your next turn, you deal an additional [[/r 1d6]]{+1d6 damage} on Strikes against the triggering enemy. The bonus increases to [[/r 2d6]]{+2d6 damage} if you use a Striking weapon or unarmed attack and [[/r 3d6]]{+3d6 damage} if you use a Striking (Major) weapon or unarmed attack.</p>"
```

Comment:
Could do an effect for this.

Reply:
I wasn't sure if it was worth it - it lasts for a small amount of time, and only against a specific enemy (and so needs a toggle). I'll add in on with the next ancestry PR.